### DISCUSSION TITLE
Why Does WiX v4 Not Detect an Upgrade?

### DISCUSSION THREAD

**Discussion by cornonthecobbs** [wixtoolset/issues#8098] [Questions]
In it's current state my WiX Installer has a Bundle project, with a Standard Bootstrapper Application, that references a single MSI project.

Once the Bundle is ran and installed, I bump the version of both projects from "1.0.0.0" to "1.0.1.0", build the Bundle project, and run the exe again.

For some reason WiX is unable to detect that this new exe is an upgrade. The log file shows that the Bundle and Package are both detected, but 'WixStdBAUpdateAvailable' evaluates to false. What am I missing?

Here is the log:
![Major Upgrade Not Working Log](https://github.com/wixtoolset/issues/assets/166164293/71a46716-0462-4602-b6b5-a9a9268131ac)

Here is the Bundle:
`<Bundle Name="MAS-iX" Manufacturer="Makino Milling Machine Co" Version="1.0.0.0" UpgradeCode="c7bc7601-a842-4ec0-bf94-4b5c74a89c03">
    <BootstrapperApplication>
        <bal:WixStandardBootstrapperApplication ShowVersion="true" Theme="hyperlinkLicense" ThemeFile="RtfLargeTheme.xml"/>
    </BootstrapperApplication>
    <Chain>
        <MsiPackage
              Name="Status Monitor"
              SourceFile="$(StatusMonitorInstaller.TargetPath)">
        </MsiPackage>
    </Chain>
</Bundle>`

Here is the MSI Package:
<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
<Package Name="MAS-iX Status Monitor" Manufacturer="Makino Milling Machine Co" Version="1.0.0.0" UpgradeCode="07cb3d25-a3e6-4929-bfbd-64b00c947abe">
    <MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeError)"/>

    <Feature Id="StatusMonitor">
        <ComponentGroupRef Id="StatusMonitorServiceHarvest"/>
        <ComponentGroupRef Id="StatusMonitorServiceComponents"/>
    </Feature>
</Package>

**Comment by barnson** [comment]
`WixStdBAUpdateAvailable` is true when there's an update bundle available for download. The first line of the screen shot shows there's an older that will be upgraded.

**Reply by cornonthecobbs** [reply]
I see. We only run our installers locally. What property could I use in order to disable certain things in the UI if the program is already installed?
